torch/csrc/jit/passes/quantization.cpp

```diff
@@ -675,24 +676,28 @@ graph(%self, %a, %b, %inplace):
 };
 
 // Check if `use` is an aten function of name `func_name` and if value
-// `v` is the nth argument of the function
-bool isAtenFuncNthArg(
-    Value* v,
+// `v` is the nth argument (if provided) of the function.
+bool matchAtenFuncToUse(
```

Comment:
Can we remove `Node*` and `Value` and just have `Use` here?

Reply:
Isn't the value necessary to compare this `v == use->inputs().at(n)`?

Reply:
no, you can do use.offset == n